feat: multiple tokens from env

lodmfjord opened the pull request:
Small changes to allow multiple tokens in env.

fox1t commented:
Thanks for the PR! Before merging, can you add test cases for single and multiple tokens?

lodmfjord commented:
Cool, I will. Probably during the weekend.

lodmfjord commented:
Added checks for single token (it now returns an array with one token) and multiple tokens.

Also my linter complained about  name: process.env.STORAGE_PATH! so I changed it to:
 name: process.env.STORAGE_PATH as string
 
 to get rid of the only warning.

fox1t reviewed (approved):
LGTM!

lodmfjord commented:
Merge?

dobesv commented:
Do you have a sense of when the changes might be pushed to dockerhub ?

tehKapa commented:
@all-contributors please add @lodmfjord for code

Klaitos commented:
Hello guys, this is a cool PR thanks ! But not specified in the doc this is sad, i think it should be mentioned here https://ducktors.github.io/turborepo-remote-cache/environment-variables that we can specify multiple tokens. What do you think ?

matteovivona commented:
> Hello guys, this is a cool PR thanks ! But not specified in the doc this is sad, i think it should be mentioned here https://ducktors.github.io/turborepo-remote-cache/environment-variables that we can specify multiple tokens. What do you think ?

you're absolutely right. I'll update the documentation right away!
